lint/internal/dummy/dummy.go

```diff
@@ -0,0 +1,4 @@
+package dummypkg
```

Comment:
Not sure, if this package additional package is fine for you.
In order the have a positive check, where the rename of a package actually does rename a package, even if the path of the package does not suggest this, I need such a package. To be able to actually import it, it can not be inside the `testdata` folder.
Therefore I think, that inside the `internal` folder of the `lint` package is the next best place.

Reply:
Why it cannot be in `testdata` ? Can it be in `lint/testdata/internal` ?

Reply:
All files located inside a folder called `testdata` are ignored by the Go compiler. Output of `go help test`:

> The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.

Therefore, it is not possible to import this package in the tests.
In my case (VS code), `goimports` does not find the package (if in folder `testdata`) and therefore removes the import on save.
In order to test (and show case) this special case where `package name` != `directory name` I need this package.